integration/inspektor-gadget/trace_tcpconnect_test.go

```diff
@@ -100,7 +100,7 @@ func TestTraceTcpconnect_latency(t *testing.T) {
 		StartAndStop: true,
 		ValidateOutput: func(t *testing.T, output string) {
 			expectedEntry := &tracetcpconnectTypes.Event{
-				Event:     BuildBaseEvent(ns, WithContainerImageName("docker.io/library/nginx:latest", isDockerRuntime)),
+				Event:     BuildBaseEventWithExtraK8s(ns, WithContainerImageName("docker.io/library/nginx:latest", isDockerRuntime)),
```

Comment:
For `ig` we can't add the K8s labels when we have non K8s containers

Reply:
I just also thought about it. Maybe for `integration/inspektor-gadget/` we introduce its own set of `BuildBaseEvent` and `BuildCommonData`.
In the long run we might add new K8s functionality and enrichment. Primarily this would only be supported by `kubectl-gadget`, since it by-default has connection to the K8s-API server